### ISSUE TITLE
Security updates

### ISSUE THREAD

**Issue by ADumaine** [montyanderson/stdrpc#43]
Hi,
Would you test/merge the pull requests since some of them have security issues?

Much appreciated.
Thanks!

**Comment by ADumaine** [comment]
Are you still maintaining this module?

**Comment by montyanderson** [comment]
@ADumaine Hi, I am. I'll get on it :)

**Comment by ADumaine** [comment]
Thank you for getting axios updated to 0.18.1 !

**Comment by ADumaine** [comment]
Hi,
I have been trying to update to the latest build but still am only getting 0.16.2.  I am having no luck getting npm to update to axios 0.18.1

 Did npm get updated?
```
stdrpc@1.0.0 | MIT | deps: 2 | versions: 4
ES6+ compatible, isomorphic JSON-RPC module for node and the browser.
https://github.com/montyanderson/stdrpc#readme

dist
.tarball: https://registry.npmjs.org/stdrpc/-/stdrpc-1.0.0.tgz
.shasum: fd7c2dbcffce2088e46bc5fb9fe07bca47eb25a2
.integrity: sha512-zyUJ5Djcbl9UNYlzdt88NldW6eulvDOQ2PI9+t51vye8BekOfn6DbLOO7t5Ah8VmUkJaF+5bGlvZm9usjI0/Rg==
.unpackedSize: 6.5 kB

dependencies:
axios: ^0.16.2         koa-basic-auth: ^3.0.0

maintainers:
- montyanderson <monty.linux@gmail.com>

dist-tags:
latest: 1.0.0

published a year ago by montyanderson <montydev@protonmail.com>
```

**Comment by ADumaine** [comment]
Monty,
When will you be able to update npm?

We have about 30k nodes using this package are are planning an update to be released soon.
Thanks!

**Comment by montyanderson** [comment]
@ADumaine Published `1.1.0`. Apologies for delay.

**Comment by ADumaine** [comment]
Thanks!
